mpegdash/nodes.py

```diff
@@ -390,7 +426,7 @@ def __init__(self):
 
         self.frame_packings = None                            # DescriptorType*
         self.audio_channel_configurations = None              # DescriptorType*
-        self.content_protections = None                       # DescriptorType*
+        self.content_protections = None                       # ContentProtectionComponent*
```

Comment:
One more minor thing about comment. :)

Reply:
My bad, I used IDE to rename class so this got missed.